### PR TITLE
dead link replaced with new link and improved the overall readme text

### PR DESCRIPTION
**Pull request by Wladefant**
old link cal.tum.sexy just leads to tum.sexy.

There doesn't seem to be a massive feature development, so I think that is not a problem to add the features to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,28 @@
-# Calendar-Proxy
+# TUM Calendar Proxy
 
+This is a proxy service that simplifies and enhances the iCal export from TUM Online. It allows you to:
 
-Proxy for the TUM iCal export to remove clutter from it and optimize the output
+- Shorten long lesson names, such as 'Grundlagen Betriebssysteme und Systemsoftware' → 'GBS'
+- Add locations that are recognized by Google Maps / Google Now
+- Filter out unwanted events, such as cancelled or optional ones
```
Comment (CommanderStorm) [inline README.md:7]:
```suggestion
- Filter out unwanted events, such as cancelled, duplicate or optional ones
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,28 @@
-# Calendar-Proxy
+# TUM Calendar Proxy
 
+This is a proxy service that simplifies and enhances the iCal export from TUM Online. It allows you to:
 
-Proxy for the TUM iCal export to remove clutter from it and optimize the output
+- Shorten long lesson names, such as 'Grundlagen Betriebssysteme und Systemsoftware' → 'GBS'
+- Add locations that are recognized by Google Maps / Google Now
```
Comment (CommanderStorm) [inline README.md:6]: I don't think google now is relevant any more.
What do you think about this?

```suggestion
- Add locations that are recognized by Google / Apple Maps
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,28 @@
...
 ## Development
-To run locally:
- - run `go run cmd/proxy/proxy.go` to install dependencies and run a local server for testing on port 8081
+If you want to run the proxy service locally or contribute to the project, you will need:
 
-To build a production image:
-- run `docker build -t tumcalproxy .`
-- run `docker run -p 8081:8081 tumcalproxy`
-- open http://127.0.0.1:8081 in your browser
+- Go 1.17 or higher
```
Comment (CommanderStorm) [inline README.md:14]: See https://github.com/TUM-Dev/CalendarProxy/blob/0bcf99e2a7b5ade76d75869c338e531d0ab55d63/go.mod#L3
```suggestion
- Go 1.19 or higher
```

File: README.md [hunk 4]
```diff
@@ -1,15 +1,28 @@
...
 
+This is a proxy service that simplifies and enhances the iCal export from TUM Online. It allows you to:
 
-Proxy for the TUM iCal export to remove clutter from it and optimize the output
+- Shorten long lesson names, such as 'Grundlagen Betriebssysteme und Systemsoftware' → 'GBS'
+- Add locations that are recognized by Google Maps / Google Now
+- Filter out unwanted events, such as cancelled or optional ones
 
-You can find more information on the about page: https://cal.tum.sexy/
+You can use the proxy service by visiting https://cal.tum.app/ and following the instructions there.
```
Comment (CommanderStorm) [inline README.md:9]:
```suggestion
You can use the proxy service by visiting <https://cal.tum.app/> and following the instructions there.
```

File: README.md [hunk 5]
```diff
@@ -1,15 +1,28 @@
-# Calendar-Proxy
+# TUM Calendar Proxy
 
+This is a proxy service that simplifies and enhances the iCal export from TUM Online. It allows you to:
```
Comment (CommanderStorm) [inline README.md:3]: Do you think adding a picture like on <https://cal.tum.app> would be worth it?

File: README.md [hunk 6]
```diff
@@ -1,15 +1,28 @@
...
-- open http://127.0.0.1:8081 in your browser
+- Go 1.17 or higher
+- Docker (optional)
+
+To run the service locally, follow these steps:
+
+- Clone this repository: `git clone https://github.com/tum-calendar-proxy/tum-calendar-proxy.git`
+- Navigate to the project directory: `cd tum-calendar-proxy`
+- Run the proxy server: `go run cmd/proxy/proxy.go`
+- The service will be available at http://localhost:8081
```
Comment (CommanderStorm) [inline README.md:22]:
```suggestion
- The service will be available at <http://localhost:8081>
```

File: README.md [hunk 7]
```diff
@@ -1,15 +1,28 @@
...
+
+To build a production image using Docker, follow these steps:
+
+- Build the image: `docker build -t tumcalproxy .`
+- Run the container: `docker run -p 8081:8081 tumcalproxy`
+- The service will be available at http://localhost:8081
```
Comment (CommanderStorm) [inline README.md:28]:
```suggestion
- The service will be available at <http://localhost:8081>
```

**Review by CommanderStorm — COMMENTED** [review]
Overall I like the change (good catch that the link is invalid and thanks for taking the time to write a PR 🎉 ).
I have left a few inline questions

File: README.md (hunk 2 above)
Comment (Wladefant) [reply README.md:6]: Yeah for sure

File: README.md (hunk 5 above)
Comment (Wladefant) [reply README.md:3]: yes for sure

**Comment by Wladefant** [comment]
Did all you asked. Ready for merge

File: README.md (hunk 5 above)
Comment (CommanderStorm) [reply README.md:5]: Interesting, that the "Marketing website" is not served from this repo.. 🤷🏻‍♂️

**Review by CommanderStorm — APPROVED** [review]

**Pull request merged**